### PR TITLE
Truetype pagination PR check

### PR DESCRIPTION
**Pull request by gmbecker**
# Pull Request

Fixes #nnn

### CODE REVIEW

**Comment by gmbecker** [comment]
I have read the CLA Document and I hereby sign the CLA

File: DESCRIPTION [hunk 1]
```diff
@@ -34,7 +34,7 @@ Imports:
     dplyr (>= 1.0.0),
     emmeans (>= 1.8.0),
     forcats (>= 1.0.0),
-    formatters (>= 0.5.6),
+    formatters (>= 0.5.6.9007),
```
Comment (shajoezhu) [inline DESCRIPTION:37]: 0.5.7.9000

Comment (shajoezhu) [inline DESCRIPTION:37]:
```suggestion
    formatters (>= 0.5.7.9000),
```

**Comment by Melkiades** [comment]
The error is weird. We should have the reference page for g_ipp. Maybe another name?

**Review by Melkiades — COMMENTED** [review]
Fixed broken link ;)

**Comment by shajoezhu** [comment]
lgtm!

**Review by Melkiades — APPROVED** [review]

**Pull request merged**